### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by MadVikingGod**
**Description:** 
Updates to otel-go v1.2.0
Updates misc other dependencies

**Documentation:** 

This also migrates from the deprecated collector semantic convention translator to the otel-go semconv.  This removes the collector as a dependancy.

### CODE REVIEW

File: launcher/config.go [hunk 1]
```diff
@@ -378,6 +377,8 @@ func ConfigureOpentelemetry(opts ...Option) Launcher {
 		c.logger.Debugf("configuration")
 		s, _ := json.MarshalIndent(c, "", "\t")
 		c.logger.Debugf(string(s))
+
+		fmt.Fprint(os.Stderr, string(s))
```
Comment (pragmaticivan) [inline launcher/config.go:381]: Leftover?

**Comment by hannahhoward** [comment]
+1 Bump!

**Review by codeboten — COMMENTED** [review]
@MadVikingGod please resolve the conflicts

**Review by codeboten — APPROVED** [review]

**Comment by MadVikingGod** [comment]
Closes: #72 #75 #77 #78 #79 #81

**Pull request merged**